react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+          </aside>
+        </section>
+
+        <section class="compare-slide">
+          <h4 class="slide-name">Using List Views</h4>
+          <div>
+          <div>
+            <img src="assets/images/flat-list.gif" alt="simple" />
+            <ul>
+              <li>long lists of data</li>
+              <li>renders items lazily</li>
+            </ul>
+          </div>
+          <div>
+            <img src="assets/images/scroll-view.gif" alt="simple" />
+            <ul>
+              <li>small number of things of a limited size</li>
+              <li>renders all its child components at once</li>
+            </ul>
+          </div>
+        </div>
+          <aside class="notes">
+            React Native provides a suite of components for presenting lists of data. Generally, there are two methods of listing data in a React Native app: displaying the list with map and using the FlatList component.
+The map method technically gives you more flexibility to customize your list the way you want with the React Native ScrollView component but it loads all the items every time which can be expensive.
+FlatList is a React Native component that only loads items that are currently visible on the screen and it deletes items as they go off screen which is more optimal for large amounts of data. Furthermore, it provides scrolling features by default. You don’t need to use extra components like ScrollView to make list data scrollable. The FlatList component requires two props: data and renderItem. data is the source of information for the list. renderItem takes one item from the source and returns a formatted component to render.
+If you need section support, you can use SectionList.
+          </aside>
+        </section>
+
+        <section class="several-pictures-4-slide-with-description">
+          <h4 class="slide-name">Touchables</h4>
+          <div>
+            <div class="fragment" data-fragment-index="1">
+              <img src="assets/images/TouchableOpacity.gif" alt="simple" height="350px" width="240px" />
+              <p>TouchableOpacity</p>
+            </div>
+            <div class="fragment" data-fragment-index="2">
+              <img src="assets/images/TouchableHighlight.gif" alt="simple" height="350px" width="240px"  />
+              <p>TouchableHighlight</p>
+            </div>
+            <div class="fragment" data-fragment-index="3">
+              <img src="assets/images/TouchableWithoutFeedback.gif" alt="simple" height="350px" width="240px"  />
+              <p>TouchableWithoutFeedback</p>
+            </div>
+            <div class="fragment" data-fragment-index="4">
+              <img src="assets/images/TouchableNativeFeedback.gif" alt="simple" height="350px" width="240px"  />
+              <p>TouchableNativeFeedback</p>
+            </div>
+          </div>
+          <aside class="notes">
```

Comment:
for line breaks please use <br/>
for section line breaks  <br/><br/>
and please add Strongs

Reply:
Done